doc/cygnus-ngsi/integration/orion_cygnus_spark.md

```diff
@@ -0,0 +1,216 @@
+  "type": "room",
+  "temperature": {
+    "value": 26.5,
+    "type": "Float"
+  }
+}
+EOF
+```
+
+Please refere to Orion Context Broker [official documentation](http://fiware-orion.readthedocs.io/en/latest/) for further details about it.
+
+[Top](#top)
+
+##<a name="section3"></a>Setting up Cygnus
+Regarding installation, do it from [FIWARE yum repository](https://github.com/telefonicaid/fiware-cygnus/blob/master/doc/cygnus-ngsi/installation_and_administration_guide/install_with_rpm.md). Once installed in its latest version, a Cygnus agent must be configured as follows:
+
+```
+$ cat /usr/cygnus/conf/agent_spark.conf
+cygnusagent.sources = http-source
+cygnusagent.sinks = avro-sink
+cygnusagent.channels = avro-channel
+
+cygnusagent.sources.http-source.type = http
+cygnusagent.sources.http-source.channels = avro-channel
+cygnusagent.sources.http-source.port = 5050
+cygnusagent.sources.http-source.handler = com.telefonica.iot.cygnus.handlers.NGSIRestHandler
+cygnusagent.sources.http-source.handler.notification_target = /notify
+cygnusagent.sources.http-source.handler.default_service = default
+cygnusagent.sources.http-source.handler.default_service_path = /
+
+cygnusagent.sinks.avro-sink.type = avro
+cygnusagent.sinks.avro-sink.channel = avro-channel
+cygnusagent.sinks.avro-sink.hostname = 0.0.0.0
+cygnusagent.sinks.avro-sink.port = 5051
+
+cygnusagent.channels.avro-channel.type = memory
+cygnusagent.channels.avro-channel.capacity = 1000
+cygnusagent.channels.avro-channel.transactionCapacity = 100
+```
+
+A Cygnus instance configuration file must be added too, somethinkg like:
+
+```
+$ cat /usr/cygnus/conf/cygnus_instance_spark.conf
+CYGNUS_USER=cygnus
+CONFIG_FOLDER=/usr/cygnus/conf
+CONFIG_FILE=/usr/cygnus/conf/agent_spark.conf
+AGENT_NAME=cygnus-ngsi
+LOGFILE_NAME=cygnus.log
+ADMIN_PORT=8081
+POLLING_INTERVAL=30
+```
+
+Then, it can be starte as a service:
+
+```
+$ sudo service cygnus start
+Starting Cygnus spark...                                    [  OK  ]
+```
+
+In order to test the integracion among Orion Context Broker and Cygnus, simply update any context entity and check the related notification reception in Cygnus logs:
+
+```
+$ tail -f /var/log/cygnus/cygnus.log
+...
+time=2017-01-27T13:33:28.236UTC | lvl=INFO | corr=a72db6ff-ce15-4d38-a3a3-95af2ecc73b8 | trans=a72db6ff-ce15-4d38-a3a3-95af2ecc73b8 | srv=sc_valencia | subsrv=/gardens | comp=cygnus-ngsi | op=getEvents | msg=com.telefonica.iot.cygnus.handlers.NGSIRestHandler[304] : [NGSIRestHandler] Received data ({  "subscriptionId" : "51c0ac9ed714fb3b37d7d5a8",  "originator" : "localhost",  "contextResponses" : [    {      "contextElement" : {        "attributes" : [          {            "name" : "temperature",            "type" : "centigrade",            "value" : "26.5"          }        ],        "type" : "room",        "isPattern" : "false",        "id" : "room1"      },      "statusCode" : {        "code" : "200",        "reasonPhrase" : "OK"      }    }  ]})
```

Comment:
Jejeje, I changed it anyway in https://github.com/telefonicaid/fiware-cygnus/pull/1398/commits/a2a29b0a2447e182fc08b85daf6579624ba04d5c :)